waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -124,18 +126,19 @@ npmDepsForWasp spec =
             ("helmet", "^6.0.0"),
             ("patch-package", "^6.4.7"),
             ("uuid", "^9.0.0"),
-            ("lodash", "^4.17.21"),
-            -- TODO: make optional
-            -- TODO: types for node
-            ("typescript", "^4.8.4")
+            ("lodash", "^4.17.21")
           ]
           ++ depsRequiredByPassport spec
           ++ depsRequiredByJobs spec,
       N.waspDevDependencies =
         AS.Dependency.fromList
           [ ("nodemon", "^2.0.19"),
             ("standard", "^17.0.0"),
-            ("prisma", show prismaVersion)
+            ("prisma", show prismaVersion),
+            -- TODO: types for node
+            -- TODO: make optional 
```

Comment:
Maybe you could explain this just a bit more? Smth like `TODO: Make these conditional once we allow users to choose if they want to use TS or not.`.